src/app/pages/system/general-settings/gui-form/gui-form.component.ts

```diff
@@ -179,7 +181,11 @@ export class GuiFormComponent {
 
         this.loader.open();
         this.ws.shuttingdown = true; // not really shutting down, just stop websocket detection temporarily
-        this.ws.call('service.restart', [ServiceName.Http]).pipe(
+        const obs = [this.ws.call('service.restart', [ServiceName.Http])];
+        if (uiCertificateChanged) {
+          obs.push(this.ws.call('system.general.ui_restart'));
```

Comment:
I feel like these two are either the same or `system.general.ui_restart` should replace `service.restart`. Please confirm with middleware.

Reply:
'service.restart` is called with `HTTP` param so I think it's meant to restart the `HTTP` service. `system.general_ui_restart` restarts the UI service at least from the names.

Reply:
I don't think there is such a thing as "UI service"